libgoal/system.go

```diff
@@ -72,3 +73,16 @@ func AlgorandDataIsPrivate(dataDir string) bool {
 	}
 	return !sc.SharedServer
 }
+
+// AlgorandDaemonSystemdManaged returns true if the algod process for a given data dir is managed by systemd
+// if not, algod will be managed as an indivudal process for the dir
+func AlgorandDaemonSystemdManaged(dataDir string) bool {
+	if dataDir == "" {
+		return false
+	}
+	sc, err := ReadSystemConfig(dataDir)
+	if err != nil {
+		return false
+	}
+	return sc.SystemdManaged
```

Comment:
where do we set `SystemdManaged` or `systemd_managed` ?

Reply:
In the system.json file of the data dir

Reply:
go ahead and add this to `installer/system.json` which is included in RPM and APT packages which install on systemd distros?

Reply:
Cool, I'm adding that with my next commit once I have the unit tests working